### PR TITLE
Add more color to nucypher cli output, fixes #1150

### LINKED ISSUE #1150: Ensure emitter errors are printed in red

### PR DESCRIPTION
**Pull request by theref**
**Type of PR:**
- [x] Other

**Required reviews:** 
- [x] 2

**What this does:**
Adds for color to the nucypher cli output, especially when `BadOptionUsage` are hit

### CODE REVIEW

**Review by KPrasch — APPROVED** [review]

**Review by piotr-roslaniec — APPROVED** [review]
LGTM :tada:

File: nucypher/cli/commands/ursula.py [hunk 1]
```diff
@@ -309,7 +309,7 @@ def create_character(self, emitter, config_file, json_ipc, load_seednodes=True):
             return ursula_config, URSULA
 
         except Keystore.AuthenticationFailed as e:
-            emitter.echo(str(e), color='red', bold=True)
+            emitter.errpr(str(e))
```
Comment (derekpierre) [inline nucypher/cli/commands/ursula.py:312]:
```suggestion
            emitter.error(str(e))
```

**Review by derekpierre — CHANGES_REQUESTED** [review]
misspelling of function call.

**Review by derekpierre — APPROVED** [review]
🎸

**Pull request merged**